bodhi/models/models.py

```diff
@@ -706,14 +707,28 @@ def new(cls, request, data):
                     build.package for build in data['builds']
                 ] if pkg.requirements], []))))
 
-        if not data['autokarma']:
+        if not data['autokarma'] and data['stable_karma'] == 1:
+            # We have to set autokarma column to False here.
+            caveats.append({
+                'name': 'autokarma',
+                'description': 'Auto-karma stable requests disabled. Make stable karma 1.',
+            })
+
+        elif not data['autokarma'] and data['stable_karma'] != 1:
+            # We have to set autokarma column to False here.
             del(data['stable_karma'])
             del(data['unstable_karma'])
             caveats.append({
                 'name': 'autokarma',
                 'description': 'Auto-karma stable requests disabled.',
             })
 
+        elif data['autokarma']:
+        # We have to set autokarma column to True here.
+
+        else:
+            pass # For further enhancing
+
         del(data['autokarma'])
```

Comment:
Since `autokarma` is now going to be a database column and is already defined as a boolean in `schemas.py`, I think we can simply remove this `del`, along with all of the stable/unstable logic above it completely. That way, we're initializing the update with `**data` which includes the new `autokarma` field. I don't think the `caveats` above this are necessary anymore. I'm working on this locally and will have more feedback and commits shortly.